i18n/translations_test.go

```diff
@@ -0,0 +1,107 @@
+package i18n
+
+import (
+	"testing"
+
+	"github.com/nicksnyder/go-i18n/i18n/bundle"
+)
+
+var bobMap = map[string]interface{}{"Person": "Bob"}
+var bobStruct = struct{ Person string }{Person: "Bob"}
+
+var testCases = []struct {
+	id   string
+	arg  interface{}
+	want string
+}{}
+
+func init() {
+	initTestCases()
+}
+
+func initTestCases() {
+	b := bundle.New()
```

Comment:
Ah, I see. This is weird because you are reading the JSON file during init but testing other formats in the actual tests.

I would actually not try to do anything in init. Since it is just a test, it is fine to re-init the small array of test cases for each test.